client/src/pages/Admin/Publications.vue

```diff
@@ -95,7 +105,17 @@ export default {
       query: GET_PUBLICATIONS
     }
   },
+  watch: {
+    "new_publication.name": function() {
+      this.vuelidateExternalResults.new_publication.name = [];
+    }
+  },
   methods: {
+    getErrorMessageKey,
+    resetForm() {
+      this.new_publication.name = "";
+        this.$v.$reset();
```

Comment:
Is this function call necessary? If it is, is this indentation intentional?

Reply:
The indentation is indeed wrong. I'll fix.

As to the necessity of the function call, I just generally like to create a reset form function so that as more things have to be done they have a place to go.  Also makes the handle method a bit more readable. IMHO.

Reply:
No, I meant: is the call to `this.$v.$reset();` necessary inside the reset function?

Reply:
Yep, you're right that's an artifact from a previous implementation.  Removed.